downstream/assemblies/platform/assembly-controller-teams.adoc

```diff
@@ -1,24 +1,31 @@
 ifdef::context[:parent-context: {context}]
 
-[id="assembly-controller-teams"]
+:_mod-docs-content-type: ASSEMBLY
+
+[id="assembly-controller-teams_{context}"]
+
+= Teams
 
 :context: controller-teams
-= Managing teams
 
-A *Team* is a subdivision of an organization with associated users, projects, credentials, and permissions.
-Teams offer a means to implement role-based access control schemes and delegate responsibilities across organizations.
-For example, you can grant permissions to a whole team rather than to each user on the team.
+A team is a subdivision of an organization with associated users, and resources. Teams provide a means to implement role-based access control schemes and delegate responsibilities across organizations. For instance, you can grant permissions to a Team rather than each user on the team.
```

Comment:
A couple of comments: 

- I would remove the comma after "associated users"--looks like it's left over from the previous version
- Third sentence, looks like there's an errant capitalization -- "you can grant permissions to a Team"